README.md

```diff
@@ -85,12 +85,30 @@ You can also manually run `nosetests` with `coverage` (but `setup.cfg` does this
 
 Try and get as close to 100% coverage as you can.
 
+
 ## Endpoints
 
 
 | Request Type | Endpoint | Description |
 |-------| ------ |------------|
 |```POST``` |```/promotions/cancel/<id>``` | Cancels a promotion based on its ID |
+|```GET``` |```/promotions``` | List all promotions or query based on certain parameters. |
+
+#### Query Parameters
+| Parameter |Description |
+|--------|-------------|
+|```amount``` | Filter the results based on the amount of discount |
+|```is_site_wide``` | Filter the results based on whether the promotion is available for the entire store |
+|```promo_code``` | Filter the results for a particular Promo Code |
+|```promo_type``` | Filter the results for a particular Promo Type |
+|```start_date``` | Filter the results based on a start date. |
+|```end_date``` | Filter the results based on an end date. |
+|``` duration``` | Filter the results based on the duration (in days) of a promotion. For example, filter out all the ads with duration greater than 10 days. |
+
```

Comment:
All these query parameters work, though there are some parameters missing from our original acceptance criteria in #10 (`active`, `store`, `product`), but I think at this point we can add a story for sprint 2 about those.